docs/RELEASE-NOTES.rst

```diff
@@ -1,38 +1,51 @@
-* Added support to send telemetry data to F5 TEEM Server 
-   - Disable this feature optionally using configuration parameter `disable-teems:"true"`
-* Added new deployment parameter `periodic-sync-interval` for configuring the periodic sync of resources
-* Added informers for kubernetes secrets to monitor the changes in secrets
-* Added support for configmaps to monitor services in same and different namespaces.
-    - Enable this support using deployment parameter `hubmode:true`
-* Helm Chart Enhancements:
-   - Added ingressClass resource installation with charts
-* Added support for AS3 3.28
+* CIS now compatible with:
+    - Kubernetes 1.21
+    - OpenShift 4.7.13 with OpenShift SDN
+    - AS3 3.28
 
+* Added support for:
+   - Multiport Service and Health Monitor for Service Type LB in CRD mode. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/crd/serviceTypeLB>`_.
+   - iRule reference in TransportServer CRD.  Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/crd/TransportServer>`_.
+   - With networking.k8s.io/v1 ingress, Add multiple BIGIP SSL client profiles with annotation "virtual-server.f5.com/clientssl". Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
+   - CIS now monitors changes to Kubernetes Secret resource.
+   - Improved performance while processing Ingress resources.
+   - CIS in AS3 agent mode now adds default cipher groups to SSL profiles for TLS v1.3.
+   - CIS in AS3 agent mode now supports OpenShift Routes with annotations `rewrite-app-root` and `rewrite-target-url`. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/docs/config_examples/ingress/networkingV1/sample-url-rewrite-annotation.yaml>`_.
+     The user need to create ingressClass resource for CIS before upgrading to CIS v2.5.0 while using ingress.
+   - CIS now supports FIC with InfoBlox (Preview).
 
-Bug Fixes
-`````````
-* :issues: `1824` Support for ingresses.networking.k8s.io/v1.
-* Fixed rewrite-url annotation doesnt support to rewrite all child paths to specific target domain
+* New CIS deployment configuration options:
+  - --periodic-sync-interval - Configure the periodic sync of resources.
+  - --hubmode:true - Enable Support for ConfigMaps to monitor services in same and different namespaces.
+  - --disable-teems=true|false - Configure to send anonymous analytics data to F5 TEEM infrastructure.
 
-* Fixed http redirect not working when used virtual-server.f5.com/rewrite-target-url annotation with routes
-* Fixed inconsistent behaviour with CIS (Transport Server) while adding/deleting a node to cluster.
+Bugs Fixes
+``````````
+* :issues: `1824` Ingress and IngressClass objects in networking.k8s.io/v1. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
```

Comment:
Not a BUG but an RFE. Add more info - Update CIS ClusterRole to monitor networking.k8s.io/v1 with ingressClass resources and install ingressClass for CIS.